test/cli/multinode_test.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ipfs/kubo/config"
+	"github.com/ipfs/kubo/test/cli/harness"
+	"github.com/ipfs/kubo/test/cli/testutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMultipleNodes(t *testing.T) {
+	t.Parallel()
+
+	checkSingleFile := func(nodes harness.Nodes) {
+		s := testutils.RandomStr(100)
+		hash := nodes[0].IPFSAddStr(s)
+		nodes.ForEachPar(func(n *harness.Node) {
```

Comment:
`Par` ? Quèsaco ?
Should it be `ForEachPeer` ?

Reply:
nope it's "par" for "parallel", it invokes those funcs in parallel w/ goroutines